js/src/schemas.ts

```diff
@@ -270,6 +270,8 @@ export interface Dataset extends BaseDataset {
   example_count?: number;
   session_count?: number;
   last_session_start_time?: number;
+  inputs_schema_definition?: KVMap;
+  outputs_schema_definition?: KVMap;
```

Comment:
cc @jakerachleff is this the proper interface to add these fields to? Or should they be on `BaseDataset`?

Reply:
It can be on the base i believe (that way in BE)